packages/ckeditor5-dev-tests/bin/install-dependencies.sh

```diff
@@ -2,15 +2,18 @@
 
 set -e
 
-npm install mgit2 lerna@^2.0.0-rc.1 eslint-config-ckeditor5 karma-coveralls@^1.1.2
+npm install -g mgit2
```

Comment:
AFAIR Yarn will remove this dependency (during `yarn install`) because it isn't listed in `package.json`.

Reply:
In CF we have the `mgit` with its version listed in the `package.json`. Not sure if it will work here though.

Reply:
We were avoiding relying on deps in `package.json` because doing `npm i` and then `lerna bootstrap` would install the same deps twice (so it'd be slow). That's why we had the deps listed here. I think it's still a thing here because e.g. `karma-coverals` is not needed in all the packages in our local dev envs. It's only needed on CI. And it's the opposite with the other dependencies – you don't want to install them from npm.